lib/config/config-file.js

```diff
@@ -19,7 +19,8 @@ var debug = require("debug"),
     ConfigOps = require("./config-ops"),
     validator = require("./config-validator"),
     Plugins = require("./plugins"),
-    resolveModule = require("resolve"),
+    Module = require("module"),
```

Comment:
It's a straggler I forgot to remove. `module` is how modules are implemented in Node.js: https://github.com/nodejs/node/blob/master/lib/module.js